doc/api/process.md

```diff
@@ -3343,6 +3342,33 @@ In custom builds from non-release versions of the source tree, only the
 `name` property may be present. The additional properties should not be
 relied upon to exist.
 
+## `process.execve(file[, args[, env]])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `file` {string} The name or path of the executable file to run.
```

Comment:
Nit... not in this PR but as a separate follow on we should likely align this with the fs module handling of paths so that `file` can be represented using a `Buffer` or `URL` also.